raaghu-components/src/rds-comp-invoice/rds-comp-invoice.tsx

```
@@ -1,20 +1,99 @@
+            address: value,
+        }));
+    };
+
+    const emitSaveData = (e: any) => {
+        e.preventDefault();
+        setInputReset(true);
+        setInvoiceData({
+            leagalName: "",
+            address: "",
+        })
+    }
```

ESLint

[error] Missing semicolon.

(semi)